### PR TITLE
Add task for new monitoring script

### PR DESCRIPTION
**Pull request by lego963**
Resolves: #195

### CODE REVIEW

File: playbooks/scenario4_setup.yml [hunk 1]
```diff
@@ -83,3 +83,6 @@
 
     - name: Start monitoring server
       shell: "nohup /usr/bin/python3 -m csm_test_utils as_monitor --telegraf http://localhost &" # noqa 301 305
+
+    - name: Start loadbalancer monitoring
+      shell: "nohup /usr/bin/python3 -m csm_test_utils as_load --telegraf http://localhost --target {{ lookup('env', 'LOADBALANCER_PUBLIC_IP') }} &" # noqa 301 305
```
Comment (anton-sidelnikov) [inline playbooks/scenario4_setup.yml]: add  204

**Comment by anton-sidelnikov** [comment]
Need to check before merge

File: playbooks/files/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ocomone >= 0.3.3
-csm-test-utils >= 0.2.1
+csm-test-utils >= 0.2.4
```
Comment (outcatcher) [inline playbooks/files/requirements.txt]: Should be 0.2.5

File: playbooks/scenario4_setup.yml (hunk 1 above)
Comment (outcatcher) [reply playbooks/scenario4_setup.yml]: or just make a multiline statement
or move `nohup /usr/bin/python3 -m csm_test_utils` to variables
and `{{ lookup('env', 'LOADBALANCER_PUBLIC_IP') }} ` to variables too

**Pull request merged**